### DISCUSSION TITLE
blog/00001-started-astro-github-blog/

### DISCUSSION THREAD

**Discussion by giscus** [YieumYoon/YieumYoon.github.io#2] [Announcements]
# blog/00001-started-astro-github-blog/

Astro 깃허브 블로그 시작, Github를 선택한 이유, Astro를 프레임워크로 선택한 이유

https://yieumyoon.github.io/blog/00001-started-astro-github-blog/

**Comment by dtoecho** [comment]
안녕하세요!

저도 astro로 블로그 만들려고 검색중인데요!

댓글 기능을 어떻게 구현하신건지 

괜찮으시면 정보공유 부탁드려도 될까요?

**Reply by YieumYoon** [reply]
와! 안녕하세요! 댓글 기능은 [giscus](https://giscus.app)를 이용했어요. giscus링크에 설명 나온데로 따라서 하면 금방 사용 가능해요. 다만 댓글 기능 위치는 .astro 레이아웃중 어디에 넣을지는 직접 레이아웃 파일에 넣어서 run dev 실행해서 보면서 결정했어요. 제가 사용한 테마에서는 ArticleBottomLayout.astro에 넣으니까 위치가 괜찮더라고요. 만약 더 자세한 설명이 필요하시면 간단히 글 올려드릴게요 😊

**Reply by dtoecho** [reply]
감사합니다
